components/ResourceHub/community-resources.json

```diff
@@ -197,6 +197,21 @@
       "image": "https://i.ytimg.com/vi/v_V4Wqcn7vE/maxresdefault.jpg"
     }
   },
+  {
+    "name": "Staying safe with Safe",
+    "url": "https://ackeeblockchain.com/blog/staying-safe-with-safe/",
+    "type": "Blog Post",
+    "date": "2024-03-15",
+    "abstract": "Since we audited Safe version 1.4.0, we realized its importance in the ecosystem. Here is the list of tips and resources for those who decide to go deeper.",
+    "tags": [
+      "Safe Smart Account"
```

Comment:
Should we add something like "security"?

Reply:
Good call!